index.html

```diff
@@ -0,0 +1,151 @@
+            </nav>
+        </section>
+        <!-- end of navbar -->
+
+        <!-- hero -->
+        <section class="hero-box">
+            <div class="small-headline">"Hello! Sharing World"</div>
+            <h1 class="headline">GET YOUR JS SOLUTIONS <br>2022</h1>
+            <p class="para">JS SOLUTIONS is a YouTube channel and blogging website where you can
+                get JavaScript related solutions <br>
+                I intend to create videos and write blogs about Responsive Web Designs, Javascript Programmatic Solutions, Firebase CRUD, <br> Reactjs and many more. 
+            </p>
+            <br>
+            <h2 class="date-time">
+                2015.10.15(THU) ~ 16(FRI)
+            </h2>
+            <div class="location">
+                @ FEDERAL URDU UNIVERSITY GULSHAN CAMPUS
+            </div>
+        </section>
+        <!-- end of hero -->
+
+        <!-- main program -->
+        <section class="main-program">
+            <h3 class="heading">Main Program</h3>
+            <div class="underline"></div>
+            <div class="cards-box">
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-chalkboard-user fa-2xl"></i></span>
+                    <h5 class="card-heading">Lecture</h5>
+                    <p class="card-text">Listen to the speakers about how you can master web development</p>
+                </div>
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-blog fa-2xl"></i></span>
+                    <h5 class="card-heading">Tech Blogs</h5>
+                    <p class="card-text">Read blogs on coding tips and tricks to enhance your knowledge</p>
+                </div>
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-video fa-2xl"></i></span>
+                    <h5 class="card-heading">Video Courses</h5>
+                    <p class="card-text">Watch high quality tutorials to master the concepts of programming</p>
+                </div>
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-message fa-2xl"></i></span>
+                    <h5 class="card-heading">Forum</h5>
+                    <p class="card-text">Have the time to share your thoughts and opinions with experts for each topic</p>
+                </div>
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-circle-question fa-2xl"></i></span>
+                    <h5 class="card-heading">Ask Questions</h5>
+                    <p class="card-text">Have a question? Get in touch directly with our experts instructors</p>
+                </div>
+            </div>
+            <a class="see-program" href="#">SEE THE WHOLE PROGRAM</a>
+            <a class="join-community" href="#">JOIN THE COMMUNITY</a>
+        </section>
```

Comment:
- [ ] Kindly add a background image to this section please check the Behance design